.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,10 +16,24 @@ If there is no corresponding issue number, fill in N/A where it says [fill_in_nu
   - This lets reviewers restart your CircleCI tests for you.
 - [ ] The PR is made from a branch that's **not** called "develop".
 
+## Proof that changes are correct
+
+<!--
+Add videos/screenshots of the user-facing interface to demonstrate that the changes made in this PR works correctly.
+-->
+Steps followed to test the changes:
```

Comment:
This is not needed if you have a video, which should be self-explanatory.

Reply:
Done!